packages/core/src/components/graphs/gauge.ts

```diff
@@ -359,7 +365,7 @@ export class Gauge extends Component {
 				"class",
 				status !== null ? `gauge-delta-arrow status--${status}` : ""
 			)
-			.attr("fill", () => (status === null ? "currentColor" : null))
+			.style("fill", () => (status === null ? "currentColor" : null))
```

Comment:
what's `currentColor` do?

Reply:
I'm not sure, I haven't found the implicit connection between `currentColor` and other stuff. I didn't touch this part in this PR.

Reply:
I removed this line since it doesn't make any difference and the color is assigned by the classname.

Reply:
could you remove the line and test whether the graph still fully functions?

Reply:
I did in 1a11c07